src/v3/src/components/Form/Form.tsx

```diff
@@ -92,9 +95,8 @@ const Form: FunctionComponent<{
     <form
       noValidate
       onSubmit={handleSubmit}
-      className="o-form" // FIXME update page objects using .o-form selectors
+      className={classes} // FIXME update page objects using .o-form selectors
       data-se="o-form"
-      style={{ maxWidth: '100%', wordBreak: 'break-word' }}
```

Comment:
This should also resolve CSP violations.

Reply:
Interestingly I didn't see any CSP violations for this when I was working on my ticket 🤔

Reply:
Oh weird 🤔 maybe its only an issue when you use javascript to manipulate the dom/style? Or maybe not lol I know in v2 this was an issue but maybe because we use Odyssey its a non issue?